app/marketing/management/commands/assemble_leaderboards.py

```diff
@@ -43,19 +61,91 @@ def default_ranks():
         dict: A nested dictionary mapping of all default ranks with empty dicts.
 
     """
-    times = ['all', 'weekly', 'quarterly', 'yearly', 'monthly']
-    breakdowns = ['fulfilled', 'all', 'payers', 'earners', 'orgs', 'keywords', 'tokens']
     return_dict = {}
-    for time in times:
-        for bd in breakdowns:
-            key = f'{time}_{bd}'
+    for time in TIMES:
+        for breakdown in BREAKDOWNS:
+            key = f'{time}_{breakdown}'
             return_dict[key] = {}
-
     return return_dict
 
+
 ranks = default_ranks()
 counts = default_ranks()
 
+def profile_to_location(handle):
```

Comment:
@timborden lol you missed one 😅

Reply:
So I think these comments (there's more then one in this file) are not reporting correctly, which is why I resolved the conversation in GitHub. Originally, there were two lines between the methods in this file. While trying to resolve the **stickler-ci** comments, I (incorrectly) tried one line. I reverted back to two lines, which appears to be inline with what Google's python style guide - https://github.com/google/styleguide/blob/gh-pages/pyguide.md#35-blank-lines but **stickler-ci** didn't notice the change. It may not have been noticed because I squashed the linting fix commits into one commit and force pushed to my branch. :man_shrugging: